manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: MongoDB,Inc.MongoDBAtlasCLI
+PackageVersion: 1.5.1
+PackageLocale: en-US
+Publisher: MongoDB, Inc.
+PackageName: MongoDB Atlas CLI

```

Comment:
```suggestion
Publisher: MongoDB
PublisherUrl: https://www.mongodb.com/
PublisherSupportUrl: https://www.mongodb.com/contact
PrivacyUrl:   https://www.mongodb.com/legal/privacy-policy
PackageName: Atlas CLI
PackageUrl: https://www.mongodb.com/tools/atlas-cli
```

Reply:
I believe PackageName and Publisher should match Apps&Features/Control Panel, and the author has put the values accordingly

Reply:
@gssbzn Can you mark this convo as resolved? This is blocking the PR from being merged.
And if you don't have the ability to do that, @stephengillie I think the solution would be to delete this review comment (or manually merge)

Reply:
@gssbzn Can you mark this convo as resolved? This is blocking the PR from being merged.
(or @stephengillie can delete the review comment to unblock the automerge)